index.js

```diff
@@ -138,21 +138,34 @@ function init (options) {
 
   options = options || {}
 
-  // There is probably 99% chance that the project root directory in located
-  // above the node_modules directory
-  var base = nodePath.resolve(
-    options.base || nodePath.join(__dirname, '../..')
-  )
-  var packagePath = base.replace(/\/package\.json$/, '') + '/package.json'
-
-  try {
-    var npmPackage = require(packagePath)
-  } catch (e) {
-    // Do nothing
+
+  var candidatePackagePaths
+  if(options.base) {
+    candidatePackagePaths = [options.base.replace(/\/package\.json$/, '')]
```

Comment:
Please wrap the `options.base.replace(/\/package\.json$/, '')` with `nodePath.resolve`, so it will fix the slashes to be cross-platform